<?xml version="1.0" encoding="utf-8"?>
<tests>
  <compliant>
    <username>hardcoded</username>
    <environment>DB_USER=hardcoded</environment>
    <MasterUserPassword>!Ref 'MasterUserPassword'</MasterUserPassword>
    <db_password>null</db_password>
  </compliant>
  <noncompliant>
    <pwd>hardcoded1</pwd>
    <passwd>hardcoded2</passwd>
    <password>hardcoded3</password>
    <inlist>DB_PASSWORD=hardcoded4</inlist>
  </noncompliant>
</tests>